movieapp/src/containers/AddEditMovieForm/AddEditMovieForm.tsx

```diff
@@ -38,6 +39,10 @@ export function AddEditMovieForm(props: MovieFormProps): JSX.Element {
           {t(CONSTANTS.LABEL_DESCRIPTION)}:
           <input type="text" id={'description'} defaultValue={props.currentValue.description} />
         </label>
+        <label htmlFor="description">
```

Comment:
if u wrap with label inner input - no need to add htmlFor as far as I remember

Reply:
fixed